### ISSUE TITLE
Handle osz files

### ISSUE THREAD

**Issue by UnderSampled** [fluddokt/FKGdxOpsuLaunchers#5]
It should be reasonably simple to add Opsu as the default app for loading .osz files, to allow direct useage from the osu website. I looked a little into it, and remember thinking that the more difficult thing will be handing the transfer of the downloaded file into the import directory.

**Comment by kairusds** [comment]
bruh that's not how that works on android

**Comment by UnderSampled** [comment]
> bruh that's not how that works on android

Yes, it is: https://www.tutlane.com/tutorial/android/android-intent-filters-with-examples

**Comment by kairusds** [comment]
oh i thought you meant unarchiving the osz file after it was downloaded automatically like osu direct

**Comment by UnderSampled** [comment]
In theory, that's what the development would be. The application can have an intend that handles the file, but the application itself would then be responsible for extracting it. It would be closer to the way Osu! works on the desktop when you download from the website.

**Comment by kairusds** [comment]
i think you'd need the app to be open for that to work but yeah it might be possible with a little hack

**Comment by kairusds** [comment]
oh this might actually work https://stackoverflow.com/a/2062112
